packages/bsky/src/services/feed/index.ts

```diff
@@ -454,3 +462,44 @@ export class FeedService {
     )
   }
 }
+
+function truncateUtf8(str: string | null | undefined, length: number) {
+  if (!str) return str
+  const encoder = new TextEncoder()
+  const utf8 = encoder.encode(str)
+  if (utf8.length > length) {
+    const decoder = new TextDecoder('utf-8', { fatal: false })
+    const truncated = utf8.slice(0, length)
+    return decoder.decode(truncated).replace(/\uFFFD$/, '')
+  }
+  return str
+}
```

Comment:
I _think_ we can get rid of this now that all clients are compatible with the lex refactor.

Reply:
ah cool cool checks out 👍 

i couldn't quite remember when we'd added this in & erred on the side of caution